src/openrct2-ui/windows/Ride.cpp

```diff
@@ -209,159 +209,159 @@ constexpr int32_t RCT1_LIGHT_OFFSET = 4;
-    { WWT_LABEL,            1, 21,      149,    61,     72,     0xFFFFFFFF,                                 STR_NONE                                        },
-    { WWT_LABEL,            1, 21,      149,    75,     86,     STR_LIFT_HILL_CHAIN_SPEED,                  STR_NONE                                        },
-    { WWT_DROPDOWN,         1,  7,      308,    47,     58,     0xFFFFFFFF,                                 STR_SELECT_OPERATING_MODE                       },
-    { WWT_BUTTON,           1,  297,    307,    48,     57,     STR_DROPDOWN_GLYPH,                         STR_SELECT_OPERATING_MODE                       },
-    { WWT_DROPDOWN,         1,  87,     308,    109,    120,    0xFFFFFFFF,                                 STR_NONE                                        },
-    { WWT_BUTTON,           1,  297,    307,    110,    119,    STR_DROPDOWN_GLYPH,                         STR_NONE                                        },
-    { WWT_LABEL,            1,  21,     149,    89,     100,    STR_NUMBER_OF_CIRCUITS,                     STR_NUMBER_OF_CIRCUITS_TIP                      },
-      SPINNER_WIDGETS      (1,  157,    308,    89,     100,    STR_NUMBER_OF_CIRCUITS_VALUE,               STR_NONE), // NB: 3 widgets
+    MakeSpinnerWidgets({157,  61}, {152, 12}, WWT_SPINNER,  1, STR_ARG_18_STRINGID                                                                 ), // NB: 3 widgets
+    MakeSpinnerWidgets({157,  75}, {152, 12}, WWT_SPINNER,  1, STR_LIFT_HILL_CHAIN_SPEED_VALUE                                                     ), // NB: 3 widgets
+    MakeWidget        ({  7, 109}, { 80, 12}, WWT_CHECKBOX, 1, STR_WAIT_FOR,                           STR_WAIT_FOR_PASSENGERS_BEFORE_DEPARTING_TIP),
+    MakeWidget        ({  7, 124}, {302, 12}, WWT_CHECKBOX, 1                                                                                      ),
+    MakeWidget        ({  7, 139}, {150, 12}, WWT_CHECKBOX, 1, STR_MINIMUM_WAITING_TIME,               STR_MINIMUM_LENGTH_BEFORE_DEPARTING_TIP     ),
+    MakeSpinnerWidgets({157, 139}, {152, 12}, WWT_SPINNER,  1, STR_ARG_10_STRINGID                                                                 ), // NB: 3 widgets
+    MakeWidget        ({  7, 154}, {150, 12}, WWT_CHECKBOX, 1, STR_MAXIMUM_WAITING_TIME,               STR_MAXIMUM_LENGTH_BEFORE_DEPARTING_TIP     ),
+    MakeSpinnerWidgets({157, 154}, {152, 12}, WWT_SPINNER,  1, STR_ARG_14_STRINGID                                                                 ), // NB: 3 widgets
+    MakeWidget        ({  7, 169}, {302, 12}, WWT_CHECKBOX, 1, STR_SYNCHRONISE_WITH_ADJACENT_STATIONS, STR_SYNCHRONISE_WITH_ADJACENT_STATIONS_TIP  ),
+    MakeWidget        ({ 21,  61}, {129, 12}, WWT_LABEL,    1                                                                                      ),
+    MakeWidget        ({ 21,  75}, {129, 12}, WWT_LABEL,    1, STR_LIFT_HILL_CHAIN_SPEED,              STR_NONE                                    ),
```

Comment:
```suggestion
    MakeWidget        ({ 21,  75}, {129, 12}, WWT_LABEL,    1, STR_LIFT_HILL_CHAIN_SPEED,                                                          ),
```

Reply:
Good catch. Including the removal of the trailing comma, we get:
```suggestion
    MakeWidget        ({ 21,  75}, {129, 12}, WWT_LABEL,    1, STR_LIFT_HILL_CHAIN_SPEED                                                           ),
```